### ISSUE TITLE
Any plans to release v5?

### ISSUE THREAD

**Issue by arthurwolf** [lllyasviel/style2paints#239]
The previews you posted are amazing, and a lot of people need/want to use it.

Do you have any idea when they will be released?

Thanks a lot.

**Comment by 444butter** [comment]
it's probably dead tbh

**Comment by arthurwolf** [comment]
Very sad. I manage somewhat with stable-diffusion but I would really like to try this out...

**Comment by 444butter** [comment]
yes i have been looking for a manga colorizer for a while, with all the ai it seems like there would be one

**Comment by arthurwolf** [comment]
I mean, stable diffusion will work as a colorizer.

Use a drawing/manga/anime/watercolor/etc model, use a controlnet to do lineart coloring (see tutorials on the internet), get the final image, and merge it with the original with a multiply operation, and you'll have a pretty ok coloring.

![image](https://github.com/user-attachments/assets/c80756b1-dfe2-4ef6-8678-a6b8511a2534)

In this example I randomize the parameters a lot to have a wide choice but you can also finetune things much more.

What do you need the manga colorizer for? I'm working on a manga-to-anime pipeline.

**Comment by 444butter** [comment]
there was a animated music video and i thought it would be fun to try and colorize it. also thanks for showing me it is still possible

**Comment by eeeeeeeeeee273** [comment]
https://discord.com/invite/hZRX9KDq
Theres this discord ai manga color they different apps or style2paints and inkhue dunno i forgot they use like 2 apps

**Comment by 444butter** [comment]
thanks

**Comment by 444butter** [comment]
> https://discord.com/invite/hZRX9KDq Theres this discord ai manga color they different apps or style2paints and inkhue dunno i forgot they use like 2 apps

thanks

**Comment by My12123** [comment]
@eeeeeeeeeee273 The link has expired. How to use your offer.

**Comment by eeeeeeeeeee273** [comment]
> [@eeeeeeeeeee273](https://github.com/eeeeeeeeeee273) The link has expired. How to use your offer.

https://discord.gg/DE2MP5DK
